core/src/test/java/org/hibernate/ogm/backendtck/associations/onetoone/GridDialectOperationInvocationForOneToOneTest.java

```diff
@@ -15,16 +15,25 @@
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.ogm.cfg.OgmProperties;
+import org.hibernate.ogm.dialect.batch.spi.BatchableGridDialect;
 import org.hibernate.ogm.dialect.impl.GridDialects;
+import org.hibernate.ogm.dialect.spi.DuplicateInsertPreventionStrategy;
 import org.hibernate.ogm.dialect.spi.GridDialect;
+import org.hibernate.ogm.model.impl.DefaultEntityKeyMetadata;
+import org.hibernate.ogm.utils.GridDialectType;
 import org.hibernate.ogm.utils.InvokedOperationsLoggingDialect;
 import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.SkipByGridDialect;
 
 import static org.fest.assertions.Assertions.assertThat;
 
 /**
  * @author Emmanuel Bernard emmanuel@hibernate.org
+ * @author Guillaume Smet
  */
+@SkipByGridDialect(value = { GridDialectType.CASSANDRA, GridDialectType.NEO4J_REMOTE, GridDialectType.NEO4J_EMBEDDED, GridDialectType.REDIS_HASH },
+		comment = "For Cassandra and Neo4j, the getAssociation always return an association, thus we don't have the createAssociation call. " +
```

Comment:
Hum, interesting, I was not aware of this different behavior. Not sure about the implication :/